draft-ietf-anima-brski-prm.md

```diff
@@ -520,127 +540,148 @@ In {{!RFC8995}}, pledges instead need to continuously request enrollment from a
+A registrar with combined BRSKI and BRSKI-PRM functionality MAY detect if the bootstrapping is performed by the pledge directly (BRSKI case) or by a Registrar-Agent (BRSKI-PRM case) based on the utilized credentials for client authentication during the TLS session establishment and switch the operational mode from BRSKI to BRSKI-PRM.
+
+This may be supported by a specific naming in the SAN (subject alternative name) component of the Registrar-Agent EE certificate. TODO(totally unclear purpose)
+
+Alternatively, this may be supported by using an EE certificate signed by the domain owner for the client authentication of the Registrar-Agent. TODO(statement in l.602 already relies on this EE cert, what is the alternative here?)
+Using an EE certificate also allows the registrar to verify that a Registrar-Agent is authorized to perform the bootstrapping of a pledge. TODO(yes, this is why there is an RegAgt EE cert in the first place; what is the point here?)
+See also Agent Proximity Assertion in {{agt_prx}}.
 
-This may be supported by a specific naming in the SAN (subject alternative name) component of the EE certificate of the Registrar-Agent.
+Using an EE certificate for TLS client authentication of the Registrar-Agent is a deviation from {{!RFC8995}}, in which the pledge IDevID certificate is used to perform TLS client authentication. TODO(again l.602 is already based on this fact.)
 
-Alternatively, this may be supported by using an LDevID certificate signed by the domain owner for the client authentication of the Registrar-Agent.
-Using an LDevID certificate also allows the registrar to verify that a Registrar-Agent is authorized to perform the bootstrapping of a pledge.
-See also agent-proximity assertion in {{agt_prx}}.
 
-Using an LDevID certificate for TLS client authentication of the Registrar-Agent is a deviation from {{!RFC8995}}, in which the IDevID credential of the pledge is used to perform TLS client authentication.
 
+## Registrar-Agent {#agent_component}
 
-## Registrar-Agent
+The Registrar-Agent is a new component in BRSKI-PRM that provides a store and forward communication path with secure message passing between pledges in responder mode and the domain registrar.
 
-The Registrar-Agent is a new component in BRSKI-PRM that provides a secure message passing service between pledges in responder mode and the domain registrar.
+It uses its own EE certificate and corresponding credentials (i.e., private key) for TLS client authentication and for signing agent-signed data objects.
+This EE certificate MUST include a SubjectKeyIdentifier as defined in {{Section 4.2.1.2 of !RFC5280}}, which is used as a reference in the context of the agent-signed data object as defined in {{jws-asd}}.
+Note that this is an additional requirement for issuing the Registrar-Agent EE certificate, as {{!IEEE-802.1AR}} only requires the SubjectKeyIdentifier to be included for intermediate CA certificates. TODO(I though .1AR does not apply here anyway?)
```

Comment:
This targets EE certificates that may be unrelated to 802.1AR.
Hence, I think we should remove the subclause starting with "as {{!IEEE..."

Reply:
True, I would propose to delte the note, as this is a requirement for the Registrar EE certificate, independent, of its provisioning and if it is a LDevID certificate or not.

Reply:
Removed .1AR part.